askup/utils/general.py

```diff
@@ -482,31 +495,66 @@ def get_student_last_week_incorrect_answers_count(user_id):
     return 0
 
 
+def get_user_organizations_queryset(user):
+    """
+    Return the queryset of sorted by name user organizations.
+    """
+    user = User.objects.get(pk=user) if isinstance(user, int) else user
```

Comment:
I would use a certain type of the user param

Reply:
done